src/components/mobile/mobile.vue

```diff
@@ -34,7 +34,7 @@
     </template>
 
     <template #toolbar v-if="$x.query.search">
-      <MobileToolbar class="x-padding--05 x-padding--top-00" />
+      <MobileToolbar class="x-padding--left-05 x-padding--bottom-05" />
```

Comment:
I'm not entirely sure that this is well done. The icon for the bigger grid cells is confusing in terms of spacing 🤣.

What I can see in figma is that the right spacing should be a little bigger than the left spacing:
<img width="305" alt="image" src="https://user-images.githubusercontent.com/68222542/179719873-37ecf41f-860a-4c7a-9415-3180dab80025.png">
<img width="278" alt="image" src="https://user-images.githubusercontent.com/68222542/179719929-744cea1f-e9af-48b8-b006-385dffeea6dd.png">

And if we remove the env selector, they are now equal.  Doesn't look bad IMHO, but it does not match the designs.
<img width="989" alt="image" src="https://user-images.githubusercontent.com/68222542/179720392-fb4b1942-c3c0-449f-bf0e-72faa3d86fff.png">

Reply:
Talked with Cris and we fixed it the best we could.
The thing is that the button for the column picker in the designs and in the archetype look the same but are different.

 In the designs, the column picker icon is a button with a `padding` of `8px` and a `gap` or `padding` between elements of `8px`. In the archetype, it's just a button with a `padding` of `16px`. That creates a bit of a mismatch on the right side of the last icon. Setting the `padding-right` of the toolbar to `8px` corrects the extra padding of the icon and makes it the same as the designs.